app/views/RoomsListView/index.js

```diff
@@ -311,6 +316,15 @@ export default class RoomsListView extends LoggedView {
 		}
 	}
 
+	// Just for tests purposes
```

Comment:
should it be removed or it's for e2e tests?

Reply:
It'll be removed before merge. It's just a test to see if markdown is taking too much.